src/gwt/src/org/rstudio/studio/client/RStudioGinModule.java

```diff
@@ -365,6 +366,8 @@ protected void configure()
 
       bind(DocDisplay.class).to(AceEditor.class);
       
+      bind(ChunkIconsManager.class).in(Singleton.class);
```

Comment:
Not sure that this needs to be a Singleton (is there state we need to share across invocations?).